src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -379,9 +388,8 @@ Browser password and autofill systems are not simple. The algorithms for
 guessing, storing and displaying values are not standardized, and vary from
 platform to platform. For example, as pointed out by [Hidde de
 Vries](https://hiddedevries.nl/en/blog/2018-01-13-making-password-managers-play-ball-with-your-login-form):
-"Firefox's password manager complements [its
-heuristics](https://dxr.mozilla.org/firefox/source/toolkit/components/passwordmgr/src/nsLoginManager.js#626)
-with a [recipe system](https://bugzilla.mozilla.org/show_bug.cgi?id=1119454)."
+"Firefox's password manager complements its heuristics with a 
```

Comment:
The heuristics link had become broken, and I'm not sure where I could link to now, so I removed it.